src/pretix/plugins/sendmail/views.py

```diff
@@ -78,24 +78,32 @@ def form_valid(self, form):
 
         if self.request.POST.get("action") == "preview":
             for l in self.request.event.settings.locales:
+
+                context_dict = {
```

Comment:
You can't move this out of the ``with language(l):`` block, otherwise the date formats etc. will be wrong.